platform/darwin/src/MGLGeometry.h

```diff
@@ -24,6 +24,14 @@ typedef struct __attribute__((objc_boxable)) MGLMapPoint {
     CGFloat zoomLevel;
 } MGLMapPoint;
 
+/* Defines a 4x4 matrix. */
+typedef struct MGLMatrix4 {
+    double m00, m01, m02, m03;
```

Comment:
Note that GLKMatrix4’s fields are zero-based, like this struct’s, while CATransform3D’s fields are one-based, per mathematical convention. I guess it’s fine as long as we’re consistent with _one_ of the built-in structs.